src/main/java/seedu/address/logic/commands/wallet/WalletFindCommand.java

```diff
@@ -0,0 +1,48 @@
+package seedu.address.logic.commands.wallet;
+
+import static seedu.address.logic.parser.CliPrefix.WALLET_COMMAND_TYPE;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.model.Model;
+import seedu.address.model.transaction.DescriptionContainsKeywordsPredicate;
+
+/**
+ * Finds and lists all transactions in wallet book whose name contains any of the argument keywords.
+ * Keyword matching is case insensitive.
+ */
+public class WalletFindCommand extends Command {
+    public static final String COMMAND_WORD = "find";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all transactions whose names contain any of "
```

Comment:
I think shouldn't use the word 'whose' since transaction is not person, but at the same time i also don't really know how to phrase it haha. which? with?

Reply:
wah cheyanne grammar nazi
I think whose is fine HAHA

Reply:
how about 
`Finds all transactions with description containing any of... `
HAHA